recipes/taocpp-taopq/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+import os.path
+
+from conans import ConanFile, CMake
+
+
+class CppsortTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        bin_path = os.path.join("bin", "test_package")
```

Comment:
Indent with `tools.cross_building`

Reply:
ctrl-c ctrl-v curse. Thanks for reviewing!

Reply:
No problem. Looking at code is easier then writing :smile: